### PR TITLE
completed the tasks

### PR DESCRIPTION
**Pull request by erdemkoral**
#### Single-line Summary
**Today, Andrew and  I paired together. It took about 2hours

#### Reflect and summarize on your process for each `TODO` item :  
  1. First, we started on making the ajax call function
  2. Next, we worked on passing the data  html.
  3. We figured the stretch goal but run out of time.

#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] We have titled the Pull Request similar to our branch name (ex: 'brian-rick'). 
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- [x] There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

File: starter-code/scripts/article.js [hunk 1]
```diff
@@ -50,13 +48,19 @@ Article.fetchAll = function() {
...
     // TODO: When we don't already have the rawData,
     // we need to retrieve the JSON file from the server with AJAX (which jQuery method is best for this?),
     // cache it in localStorage so we can skip the server call next time,
     // then load all the data into Article.all with the .loadAll function above,
     // and then render the index page.
+    articleView.initIndexPage();
+    $.getJSON( 'data/hackerIpsum.json', function( json ) {
+      console.log( 'JSON Data: ', json);
+      localStorage.setItem('rawData', JSON.stringify(json));
+    });
```
Comment (JPGothro) [inline starter-code/scripts/article.js]: When I run your code, no articles are loaded into the page. There is now data in localStorage, but nothing shows up in the browser. Can you tell me why?
.
.
.
.
As a clue, you are invoking the articleView.initIndexPage function in both the "IF" and "ELSE", why not put that line after the IF/ELSE statement so it is always executed when you are done handling data?

**Comment by erdemkoral** [comment]
It only runs on live server, I guess

**Comment by JPGothro** [comment]
If you want me to re-grade the assignment, please re-submit it in Canvas. 
Thanks.

**Comment by erdemkoral** [comment]
fixed errors

**Comment by kbrumer** [comment]
👍

**Pull request closed without merging**